src/btree/bt_read.c

```diff
@@ -183,20 +183,14 @@ __las_page_instantiate(WT_SESSION_IMPL *session, WT_REF *ref)
 		/* Allocate the WT_UPDATE structure. */
 		WT_ERR(cursor->get_value(
 		    cursor, &las_txnid, &las_timestamp,
-		    &prepare_state, &upd_type, &las_value));
+		    &durable_timestamp, &prepare_state, &upd_type, &las_value));
 		WT_ERR(__wt_update_alloc(
 		    session, &las_value, &upd, &incr, upd_type));
 		total_incr += incr;
 		upd->txnid = las_txnid;
 		upd->timestamp = las_timestamp;
 		upd->prepare_state = prepare_state;
-		/*
-		 * Use the commit timestamp as the durable timestamp, since
-		 * non durable committed updates don't currently get written to
-		 * lookaside, so the two timestamps should always be identical.
-		 */
-		if (prepare_state != WT_PREPARE_INPROGRESS)
-			upd->durable_timestamp = upd->timestamp;
+		upd->durable_timestamp = durable_timestamp;
```

Comment:
Did you remove the conditional setting of the durable timestamp for a reason?

Reply:
Yes - per discussion with Vamsi this check is no longer required. With this change, for every update that we write to the LAS table, we also write the durable timestamp. Therefore at the time of LAS read, it can be read unconditionally.